src/cellRenderers/CellProvider.js

```diff
@@ -234,10 +245,23 @@ var CellProvider = Base.extend('CellProvider', {
         config.minWidth = config.minWidth + 2 * (iconWidth);
     },
 
-    renderMultiLineText: function(gc, x, y, height, width, config, val) {
+    /**
+     * @summary Renders multiline text
+     * @param {CanvasGraphicsContext} gc
+     * @param {number} config.bounds.x - the x screen coordinate of my origin
+     * @param {number} config.bounds.y - the y screen coordinate of my origin
+     * @param {number} config.bounds.width - the width I'm allowed to draw within
```

Comment:
Please use whole sentences with initial capital letter and final period. I know this sounds peevish but in fact it otherwise looks lousy on the rendered jsdocs page when you have what appear to be random sentence fragments all over the place.